_config.yml

```diff
@@ -2,11 +2,11 @@
 title: Cloud Information Center
 description: A site for you to learn about cloud!
 baseurl: "" # the subpath of your site, e.g. /blog/
-url: "https://federalist-modern-team-template.18f.gov" # the base hostname & protocol for your site
-logo: /assets/img/logo-main.gif
+url: "https://federalist-proxy.app.cloud.gov/preview/18f/cic-site/implement-sitemap-content/" # the base hostname & protocol for your site
```

Comment:
I'm not totally familiar with all parts of config, but it seems surprising we'd use the specific branch name here instead of ending say, after `/cic-site/`. I'd love to understand more about what this does (offline is fine!)

Reply:
yeah, i think is left over from when i was trying to debug the urls, will update